vcr/test.go

```diff
@@ -60,16 +61,22 @@ type mockContext struct {
 	tx          *network.MockTransactions
 	vcr         *vcr
 	keyResolver *types.MockKeyResolver
+	docResolver *types.MockDocResolver
 }
 
 func newMockContext(t *testing.T) mockContext {
 	testDir := io.TestDirectory(t)
 	ctrl := gomock.NewController(t)
 	crypto := crypto.NewMockKeyStore(ctrl)
 	tx := network.NewMockTransactions(ctrl)
-	keystore := types.NewMockKeyResolver(ctrl)
-	vcr := NewVCRInstance(crypto, keystore, tx).(*vcr)
+	tx.EXPECT().Subscribe(gomock.Any(), gomock.Any()).AnyTimes()
+	keyResolver := types.NewMockKeyResolver(ctrl)
+	docResolver := types.NewMockDocResolver(ctrl)
+	vcr := NewVCRInstance(crypto, docResolver, keyResolver, tx).(*vcr)
 	vcr.trustConfig = trust.NewConfig(path.Join(testDir, "trust.yaml"))
+	if err := vcr.Configure(core.ServerConfig{Datadir: testDir}); err != nil {
+		t.Fatal(err)
+	}
 	t.Cleanup(func() {
 		ctrl.Finish()
 	})
```

Comment:
Nitpick: this can be removed the test controller will automatically do this when running `gomock.NewController` in a unittest.